k8s/namespaces/rd/rd-professional-api/aat.yaml

```diff
@@ -8,5 +8,5 @@ spec:
       environment:
         DELETE_ORG: false
         ACTIVE_ORG_EXT: true
-        PRD_S2S_AUTHORISED_SERVICES: rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,finrem_case_orchestration,fpl_case_service,iac,aac_manage_case_assignment,divorce_frontend,nfdiv_case_api,am_org_role_mapping_service,probate_backend,civil_service,payment_app,prl_cos_api,civil_general_applications
```

Comment:
@geciciserkan : Is there any requirement to whitelist special tribunals from professional reference data. If so, please let us know. For production whitelisting you have to go through the release process.

Reply:
I am not sure if we need it. I removed for now. I will add if we need it in the future